scripts/crond/back-nightly

```diff
@@ -20,7 +20,7 @@ catch_output_email ncanda-admin@sri.com "NCANDA Pipeline Feeder Messages (import
 
 # Export NP/clinical/dempgraphics data into image analysis pipeline directories
 #   This needs to come AFTER "import_mr_sessions", because otherwise we cannot get ages-at-MRI from REDCap for the export.
-catch_output_email ncanda-admin@sri.com "NCANDA Pipeline REDCap Exporter Messages (export_measures)" ${SIBIS}/scripts/redcap/export_measures --datadict-dir /fs/ncanda-share/pipeline/datadict --locked_form_report /fs/ncanda-share/pipeline/cases
+catch_output_email ncanda-admin@sri.com "NCANDA Pipeline REDCap Exporter Messages (export_measures)" ${SIBIS}/scripts/redcap/export_measures --datadict-dir /fs/ncanda-share/pipeline/datadict/redcap --locked_form_report /fs/ncanda-share/pipeline/cases
```

Comment:
@kipohl didn't we agree to call these `clinical` and `neuropsych` rather than `redcap`?